extensions/ql-vscode/src/run-remote-query.ts

```diff
@@ -47,6 +50,40 @@ export async function findLanguage(
   return language;
 }
 
+interface RepoListQuickPickItem extends QuickPickItem {
+  repoList: string[];
+}
+
+/**
+ * Gets the repositories to run the query against.
+ */
+async function getRepositories(): Promise<string[] | undefined> {
+  if (REPO_LISTS) {
```

Comment:
Perhaps better to also check that `REPO_LISTS` is not an empty object, eg-

```suggestion
  if (Object.keys(REPO_LISTS || {}).length) {
```

Also, see my comment above. This might be better:

```suggestion
  if (Object.keys(getRemoteRepositoryLists() || {}).length) {
```